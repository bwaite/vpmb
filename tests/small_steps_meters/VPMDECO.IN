TRIMIX DIVE TO 80 MSW !Description of dive
1                      !Number of gas mixes
.15,.45,.40            !Fraction O2, Fraction He, Fraction N2
1                      !Profile code 1 = descent
0,80,23,1,.7          !Starting depth, ending depth, rate, gasmix, setpoint
2                      !Profile code 2 = constant depth
80,30,1,1.3           !Depth, run time at end of segment, gasmix, setpoint
99                     !Profile code 99 = decompress
2                      !Number of ascent parameter changes
80,1,-10,1.5,1.6       !Starting depth, gasmix, rate, step size,setpoint
6,1,-10,6,1.6        !Starting depth, gasmix, rate, step size,setpoint
1                      !Repetitive code 1 = repetitive dive to follow
60                     !Surface interval time in minutes
TRIMIX DIVE TO 80 MSW !Description of dive
3                      !Number of gas mixes
.15,.45,.40            !Fraction O2, Fraction He, Fraction N2
.36,.00,.64            !Fraction O2, Fraction He, Fraction N2
1.0,.00,.00            !Fraction O2, Fraction He, Fraction N2
1                      !Profile code 1 = descent
0,80,23,1,0           !Starting depth, ending depth, rate, gasmix
2                      !Profile code 2 = constant depth
80,30,1,0             !Depth, run time at end of segment, gasmix
99                     !Profile code 99 = decompress
3                      !Number of ascent parameter changes
80,1,-10,3,0         !Starting depth, gasmix, rate, step size
33,2,-10,3,0         !Change depth, gasmix, rate, step size
6,3,-10,6,0          !Change depth, gasmix, rate, step size
0                      !Repetitive code 0 = last dive/end of file
