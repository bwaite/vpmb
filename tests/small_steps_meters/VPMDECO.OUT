                          DECOMPRESSION CALCULATION PROGRAM
                        Developed in FORTRAN by Erik C. Baker
 
 
Program Run:    03-01-2010 at 06:57 pm                       Model: VPM-B
 
Description:    TRIMIX DIVE TO 80 MSW !Description of dive                            
 
Gasmix Summary:                        FO2    FHe    FN2
                          Gasmix # 1  0.150  0.450  0.400
 
 
                                    DIVE PROFILE
 
Seg-  Segm.  Run   | Gasmix | Ascent    From     To      Rate    | Constant
ment  Time   Time  |  Used  |   or     Depth   Depth    +Dn/-Up  |  Depth
  #   (min)  (min) |    #   | Descent  (mswg)  (mswg)  (msw/min) |  (mswg)
----- -----  ----- | ------ | -------  ------  ------  --------- | --------
  1     3.5    3.5 |    1   | Descent     0.     80.      23.0   |
  2    26.5   30.0 |    1   |                                    |    80.
 
 
                               DECOMPRESSION PROFILE
 
          Leading compartment enters the decompression zone at   64.8 mswg
                 Deepest possible decompression stop is   64.5 mswg
 
Seg-  Segm.  Run   | Gasmix | Ascent   Ascent   Col   |  DECO   STOP   RUN
ment  Time   Time  |  Used  |   To      Rate    Not   |  STOP   TIME   TIME
  #   (min)  (min) |    #   | (mswg) (msw/min)  Used  | (mswg)  (min)  (min)
----- -----  ----- | ------ | ------ --------- ------ | ------  -----  -----
  3     2.3   32.3 |    1   |   57.    -10.0          |
  4     0.7   33.0 |    1   |                         |    57      1     33
  5     0.2   33.2 |    1   |   56.    -10.0          |
  6     0.8   34.0 |    1   |                         |    55      1     34
  7     0.2   34.2 |    1   |   54.    -10.0          |
  8     0.8   35.0 |    1   |                         |    54      1     35
  9     0.2   35.2 |    1   |   53.    -10.0          |
 10     0.8   36.0 |    1   |                         |    52      1     36
 11     0.2   36.2 |    1   |   51.    -10.0          |
 12     0.8   37.0 |    1   |                         |    51      1     37
 13     0.2   37.2 |    1   |   50.    -10.0          |
 14     0.8   38.0 |    1   |                         |    49      1     38
 15     0.2   38.2 |    1   |   48.    -10.0          |
 16     0.8   39.0 |    1   |                         |    48      1     39
 17     0.2   39.2 |    1   |   47.    -10.0          |
 18     0.8   40.0 |    1   |                         |    46      1     40
 19     0.2   40.2 |    1   |   45.    -10.0          |
 20     0.8   41.0 |    1   |                         |    45      1     41
 21     0.2   41.2 |    1   |   44.    -10.0          |
 22     0.8   42.0 |    1   |                         |    43      1     42
 23     0.2   42.2 |    1   |   42.    -10.0          |
 24     0.8   43.0 |    1   |                         |    42      1     43
 25     0.2   43.2 |    1   |   41.    -10.0          |
 26     0.8   44.0 |    1   |                         |    40      1     44
 27     0.2   44.2 |    1   |   39.    -10.0          |
 28     0.8   45.0 |    1   |                         |    39      1     45
 29     0.2   45.2 |    1   |   38.    -10.0          |
 30     0.8   46.0 |    1   |                         |    37      1     46
 31     0.2   46.2 |    1   |   36.    -10.0          |
 32     0.8   47.0 |    1   |                         |    36      1     47
 33     0.2   47.2 |    1   |   35.    -10.0          |
 34     1.8   49.0 |    1   |                         |    34      2     49
 35     0.2   49.2 |    1   |   33.    -10.0          |
 36     1.8   51.0 |    1   |                         |    33      2     51
 37     0.2   51.2 |    1   |   32.    -10.0          |
 38     1.8   53.0 |    1   |                         |    31      2     53
 39     0.2   53.2 |    1   |   30.    -10.0          |
 40     2.8   56.0 |    1   |                         |    30      3     56
 41     0.2   56.2 |    1   |   29.    -10.0          |
 42     1.8   58.0 |    1   |                         |    28      2     58
 43     0.2   58.2 |    1   |   27.    -10.0          |
 44     2.8   61.0 |    1   |                         |    27      3     61
 45     0.2   61.2 |    1   |   26.    -10.0          |
 46     2.8   64.0 |    1   |                         |    25      3     64
 47     0.2   64.2 |    1   |   24.    -10.0          |
 48     3.8   68.0 |    1   |                         |    24      4     68
 49     0.2   68.2 |    1   |   23.    -10.0          |
 50     4.8   73.0 |    1   |                         |    22      5     73
 51     0.2   73.2 |    1   |   21.    -10.0          |
 52     3.8   77.0 |    1   |                         |    21      4     77
 53     0.2   77.2 |    1   |   20.    -10.0          |
 54     4.8   82.0 |    1   |                         |    19      5     82
 55     0.2   82.2 |    1   |   18.    -10.0          |
 56     6.8   89.0 |    1   |                         |    18      7     89
 57     0.2   89.2 |    1   |   17.    -10.0          |
 58     6.8   96.0 |    1   |                         |    16      7     96
 59     0.2   96.2 |    1   |   15.    -10.0          |
 60     7.8  104.0 |    1   |                         |    15      8    104
 61     0.2  104.2 |    1   |   14.    -10.0          |
 62    10.8  115.0 |    1   |                         |    13     11    115
 63     0.2  115.2 |    1   |   12.    -10.0          |
 64    11.8  127.0 |    1   |                         |    12     12    127
 65     0.2  127.2 |    1   |   11.    -10.0          |
 66    14.8  142.0 |    1   |                         |    10     15    142
 67     0.2  142.1 |    1   |    9.    -10.0          |
 68    17.9  160.0 |    1   |                         |     9     18    160
 69     0.2  160.1 |    1   |    8.    -10.0          |
 70    22.9  183.0 |    1   |                         |     7     23    183
 71     0.2  183.1 |    1   |    6.    -10.0          |
 72   517.8  701.0 |    1   |                         |     6    518    701
 73     0.6  701.6 |    1   |    0.    -10.0          |
 
REPETITIVE DIVE:
 
                          DECOMPRESSION CALCULATION PROGRAM
                        Developed in FORTRAN by Erik C. Baker
 
 
Program Run:    03-01-2010 at 06:57 pm                       Model: VPM-B
 
Description:    TRIMIX DIVE TO 80 MSW !Description of dive                            
 
Gasmix Summary:                        FO2    FHe    FN2
                          Gasmix # 1  0.150  0.450  0.400
                          Gasmix # 2  0.360  0.000  0.640
                          Gasmix # 3  1.000  0.000  0.000
 
 
                                    DIVE PROFILE
 
Seg-  Segm.  Run   | Gasmix | Ascent    From     To      Rate    | Constant
ment  Time   Time  |  Used  |   or     Depth   Depth    +Dn/-Up  |  Depth
  #   (min)  (min) |    #   | Descent  (mswg)  (mswg)  (msw/min) |  (mswg)
----- -----  ----- | ------ | -------  ------  ------  --------- | --------
  1     3.5    3.5 |    1   | Descent     0.     80.      23.0   |
  2    26.5   30.0 |    1   |                                    |    80.
 
 
                               DECOMPRESSION PROFILE
 
          Leading compartment enters the decompression zone at   64.8 mswg
                 Deepest possible decompression stop is   63.0 mswg
 
Seg-  Segm.  Run   | Gasmix | Ascent   Ascent   Col   |  DECO   STOP   RUN
ment  Time   Time  |  Used  |   To      Rate    Not   |  STOP   TIME   TIME
  #   (min)  (min) |    #   | (mswg) (msw/min)  Used  | (mswg)  (min)  (min)
----- -----  ----- | ------ | ------ --------- ------ | ------  -----  -----
  3     2.3   32.3 |    1   |   57.    -10.0          |
  4     0.7   33.0 |    1   |                         |    57      1     33
  5     0.3   33.3 |    1   |   54.    -10.0          |
  6     0.7   34.0 |    1   |                         |    54      1     34
  7     0.3   34.3 |    1   |   51.    -10.0          |
  8     0.7   35.0 |    1   |                         |    51      1     35
  9     0.3   35.3 |    1   |   48.    -10.0          |
 10     1.7   37.0 |    1   |                         |    48      2     37
 11     0.3   37.3 |    1   |   45.    -10.0          |
 12     1.7   39.0 |    1   |                         |    45      2     39
 13     0.3   39.3 |    1   |   42.    -10.0          |
 14     1.7   41.0 |    1   |                         |    42      2     41
 15     0.3   41.3 |    1   |   39.    -10.0          |
 16     2.7   44.0 |    1   |                         |    39      3     44
 17     0.3   44.3 |    1   |   36.    -10.0          |
 18     2.7   47.0 |    1   |                         |    36      3     47
 19     0.3   47.3 |    1   |   33.    -10.0          |
 20     1.7   49.0 |    2   |                         |    33      2     49
 21     0.3   49.3 |    2   |   30.    -10.0          |
 22     1.7   51.0 |    2   |                         |    30      2     51
 23     0.3   51.3 |    2   |   27.    -10.0          |
 24     1.7   53.0 |    2   |                         |    27      2     53
 25     0.3   53.3 |    2   |   24.    -10.0          |
 26     3.7   57.0 |    2   |                         |    24      4     57
 27     0.3   57.3 |    2   |   21.    -10.0          |
 28     3.7   61.0 |    2   |                         |    21      4     61
 29     0.3   61.3 |    2   |   18.    -10.0          |
 30     4.7   66.0 |    2   |                         |    18      5     66
 31     0.3   66.3 |    2   |   15.    -10.0          |
 32     7.7   74.0 |    2   |                         |    15      8     74
 33     0.3   74.3 |    2   |   12.    -10.0          |
 34     9.7   84.0 |    2   |                         |    12     10     84
 35     0.3   84.3 |    2   |    9.    -10.0          |
 36    15.7  100.0 |    2   |                         |     9     16    100
 37     0.3  100.3 |    2   |    6.    -10.0          |
 38    92.7  193.0 |    3   |                         |     6     93    193
 39     0.6  193.6 |    3   |    0.    -10.0          |
 
